<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringHashGenerationClass</class>
 <widget class="QWidget" name="StringHashGenerationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StringHashGeneration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>966</width>
        <height>773</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>配置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="devtoys.qrc">:/DevToys/icon/letter.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>大写字母</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="SlideButton" name="lowerButton" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="devtoys.qrc">:/DevToys/icon/type.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>转换类型</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>十六进制</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>base64</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>输入</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pasteButton">
           <property name="text">
            <string>粘贴</string>
           </property>
           <property name="icon">
            <iconset resource="devtoys.qrc">
             <normaloff>:/DevToys/icon/粘贴.png</normaloff>:/DevToys/icon/粘贴.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openFileButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="devtoys.qrc">
             <normaloff>:/DevToys/icon/打开文件.png</normaloff>:/DevToys/icon/打开文件.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="devtoys.qrc">
             <normaloff>:/DevToys/icon/清空.png</normaloff>:/DevToys/icon/清空.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>哈希散列生成工具</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="importTextEdit">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>输出</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="2">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="1">
             <widget class="QLineEdit" name="SHA1LineEdit"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="SHA256LineEdit"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="MD5LineEdit"/>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="SHA256CopyButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="devtoys.qrc">
                <normaloff>:/DevToys/icon/复制.png</normaloff>:/DevToys/icon/复制.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="MD5CopyButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="devtoys.qrc">
                <normaloff>:/DevToys/icon/复制.png</normaloff>:/DevToys/icon/复制.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="SHA1CopyButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="devtoys.qrc">
                <normaloff>:/DevToys/icon/复制.png</normaloff>:/DevToys/icon/复制.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>SHA256 值</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>MD5 值</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>SHA1 值</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>SHA512 值</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="SHA512LineEdit"/>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="SHA512CopyButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="devtoys.qrc">
                <normaloff>:/DevToys/icon/复制.png</normaloff>:/DevToys/icon/复制.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>79</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SlideButton</class>
   <extends>QWidget</extends>
   <header>slidebutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="devtoys.qrc"/>
  <include location="C:/Users/13123/.designer/backup/devtoys.qrc"/>
 </resources>
 <connections/>
</ui>
